docs/json_rpc_api.mediawiki

```diff
@@ -0,0 +1,2333 @@
+==1. Overview==
+
+dcrd provides a [http://json-rpc.org/wiki/specification JSON-RPC] API that is
+fully compatible with the original bitcoind/bitcoin-qt.  There are a few key
+differences between dcrd and bitcoind as far as how RPCs are serviced:
+
+* Unlike bitcoind that has the wallet and chain intermingled in the same process which leads to several issues, dcrd intentionally splits the wallet and chain services into independent processes.  See the blog post [https://blog.conformal.com/btcd-not-your-moms-bitcoin-daemon/ here] for further details on why they were separated.  This means that if you are talking directly to dcrd, only chain-related RPCs are available.  However both chain-related and wallet-related RPCs are available via [https://github.com/decred/dcrwallet dcrwallet].
+
+* dcrd is secure by default which means that the RPC connection is TLS-enabled by default
+
+* dcrd provides access to the API through both [https://en.wikipedia.org/wiki/POST_%28HTTP%29 HTTP POST] requests and [https://en.wikipedia.org/wiki/WebSocket Websockets]
+
+Websockets are the preferred transport for dcrd RPC and are used by applications
+such as [https://github.com/decred/dcrwallet dcrwallet] for inter-process
+communication with dcrd.  The websocket connection endpoint for dcrd is
+<code>wss://your_ip_or_domain:9109/ws</code>.
+
+In addition to the [[#5-standard-methods|standard API]], an [[#6-websocket-methods-websocket-specific|extension API]]
+has been developed that is exclusive to clients using Websockets. In its current
+state, this API attempts to cover features found missing in the standard API
+during the development of dcrwallet.
+
+While the [[#5-standard-methods|standard API]] is stable, the
+[[#6-websocket-methods-websocket-specific|Websocket extension API]] should be considered a work in
+progress, incomplete, and susceptible to changes (both additions and removals).
+
+The original bitcoind/bitcoin-qt JSON-RPC API documentation is available at https://en.bitcoin.it/wiki/Original_Bitcoin_client/API_Calls_list
```

Comment:
We can remove this line since it was inherited and doesn't apply to Decred.

Reply:
Done